Tvl.VisualStudio.JustMyCodeToggle/JustMyCodeTogglePackage.cs

```diff
@@ -6,14 +6,17 @@ namespace Tvl.VisualStudio.JustMyCodeToggle
     using System;
     using System.ComponentModel.Design;
     using System.Runtime.InteropServices;
+    using System.Threading;
+    using Microsoft.VisualStudio;
     using Microsoft.VisualStudio.Shell;
-    using ErrorHandler = Microsoft.VisualStudio.ErrorHandler;
     using IMenuCommandService = System.ComponentModel.Design.IMenuCommandService;
+    using Task = System.Threading.Tasks.Task;
 
     [Guid(JustMyCodeToggleConstants.guidJustMyCodeTogglePackageString)]
     [PackageRegistration(UseManagedResourcesOnly = true)]
```

Comment:
I think this should be:
```cs
[PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
```

`AllowsBackgroundLoading = true` is like a master switch that allows the package to load on a b/g thread.  `PackageAutoLoadFlags.BackgroundLoad` is just a hint for that particular `ProvideAutoLoad`.